server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -370,7 +382,12 @@ def _is_valid_layout(self, arr):
 
     @requires_data
     def _validate_data_types(self):
-        if sparse.isspmatrix(self.data.X) and not sparse.isspmatrix_csc(self.data.X):
+        # The backed API does not support interogation of the underlying sparsity or sparse matrix type
+        # Fake it by asking for a small subarray and testing it.   NOTE: if the user has ignored our
+        # anndata <= 0.7 warning, and run in backed mode with a large sparse dataset, this "small"
```

Comment:
`and run in backend mode` doesn't quite parse

Reply:
it is 'backed', not 'backend'.    In the anndata API, `backed` is short-hand for file-backed. 

I'll expand on the wording.